queries.py

```diff
@@ -37,7 +37,39 @@ def get_expenditure_w_porverty_by_country_year():
         ORDER BY e.country_name, e.year
     """)
     df['decentralized_expenditure'].fillna(0, inplace=True)
+    ## just for testing (per capital is missing, underlying database changed?)
+    df["per_capita_real_expenditure"] = df.expenditure
+    df["per_capita_expenditure"] = df.expenditure
+    return df
+
+def get_expenditure_by_func_country_year():
+    query = 'SELECT l.country_name, l.year, l.admin0, l.func, l.real_expenditure '\
+        'FROM boost.expenditure_by_country_admin_func_sub_econ_sub_year as l  '\
+        'WHERE l.func in ("Education", "Health") and l.year >= 2010'
+    df = execute_query(query)
+    return df
+
+def get_edu_private_expenditure():
+    query = 'SELECT country_name, year, real_expenditure, real_pub_expenditure '\
+            'FROM '\
+            'boost.edu_private_expenditure_by_country_year '\
+            'where year >= 2010'
```

Comment:
I know we artificially limited the year to be 2010 onwards in PowerBI. I actually cannot recall why 😅 . If you can think of a good rationale we can keep it, but consider making it configurable somewhere. Otherwise, we can just remove it for simplicity.